waf.go

```diff
@@ -82,13 +82,20 @@ func NewWAF(config WAFConfig) (WAF, error) {
 			return nil, errors.New("request body limit should be bigger than 0")
 		}
 
-		if c.requestBodyLimit < c.requestBodyInMemoryLimit {
-			return nil, errors.New("request body limit should be at least the memory limit")
+		if c.requestBodyLimit > 1073741824 {
```

Comment:
nit: I don't like throwing numbers like this. Can we define a `const` with _meaning_? Can be used in both places also.

Reply:
My bad, in a previous attempt I had

```
if c.requestBodyLimit > 1073741824 { // 1GB
```
Do you still want the constant?

Reply:
I think it is useful. We can use it in other places if needed. And it does not affect performance.